File: go/general/1598. 文件夹操作日志搜集器.go
package main

func minOperations(logs []string) int {
	var n int
	for _, v := range logs {
		if v == "./" {
			continue
		} else if v == "../" {
			if n == 0 {
				continue
			} else {
				n--
			}
		} else {
			n++
		}
	}
	return n
}

File: python/general/1598. 文件夹操作日志搜集器.py
from typing import List
class Solution:
    def minOperations(self, logs: List[str]) -> int:
        n = 0
        for log in logs:
            if log == './':
                continue
            elif log == '../':
                if n == 0: continue
                else: n-=1
            else:
                n+=1
        return n